src/api/app/models/obs_factory/staging_project.rb

```diff
@@ -258,7 +258,7 @@ def testing?(openqa_jobs)
       openqa_jobs.empty? || openqa_jobs.any? { |job| job.result == 'none' }
     end
 
-    def all_passed?
+    def all_passed?(openqa_jobs)
```

Comment:
openqa_jobs should be accessible via [this method](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/obs_factory/staging_project.rb#L122).

I think the correct patch would be to drop the parameter [here](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/obs_factory/staging_project.rb#L271)

Reply:
yeah, I was obviously torn if the better callback was to use or not use a param :)

Reply:
@bgeuken, so why should do the same with [this method](https://github.com/openSUSE/open-build-service/pull/5234/files#diff-fbfbf85428ea5314b5c5f2014ddb1231R256)